crates/forge-runner/Cargo.toml

```diff
@@ -33,7 +33,7 @@ num-traits.workspace = true
 rand.workspace = true
 url.workspace = true
 blockifier.workspace = true
-cairo-vm = "0.8.1"
+cairo-vm.workspace = true
```

Comment:
Unrelated, but someone will spend additional 15 minutes when bumping deps wondering what's going on (this dep is quite important)

Reply:
I appreciate your respect for our time :)